src/Compilers/Core/Portable/WellKnownTypes.cs

```diff
@@ -619,6 +620,7 @@ internal static class WellKnownTypes
             "System.Runtime.CompilerServices.NativeIntegerAttribute",
             "System.Runtime.CompilerServices.IsExternalInit",
             "System.Runtime.InteropServices.OutAttribute",
+            "System.Runtime.CompilerServices.PreserveBaseOverridesAttribute",
```

Comment:
>"System.Runtime.CompilerServices.PreserveBaseOverridesAttribute" [](start = 12, length = 64)

I think we should limit lookup of this attribute to the core library, therefore, should move it to the set of SpecialTypes.

Reply:
Recorded in [https://github.com/dotnet/roslyn/issues/44208](https://github.com/dotnet/roslyn/issues/44208).

---
In reply to: [442335393](https://github.com/dotnet/roslyn/pull/44025#discussion_r442335393) [](ancestors = 442335393)